### ISSUE TITLE
Contact view: Same phone number two times

### ISSUE THREAD

**Issue by ovalseven8** [threema-ch/threema-web#136]
At the contact view in Threema Web it seems like there is a bug.

There you can see the same mobile phone number **two times**. The only difference between these two mobile phone numbers is that in the first one there are spaces between the parts of the number.
For example: 00 49 XXX
The second mobile phone number is without the spaces:
0049XXX

![screenshot](https://cloud.githubusercontent.com/assets/8258609/23624688/e791764c-02a6-11e7-93ea-ab9356004481.png)

## Your Environment
* Threema Web version: 1.0.5
* Threema for Android version: Up-to-date

**Comment by dbrgn** [comment]
@ovalseven8 in that case the two slightly differing numbers are probably both linked with your system contact. could you check?

**Comment by ovalseven8** [comment]
Yeah, the contact is linked with the system contact. But I have only one saved mobile phone number there.

**Comment by dbrgn** [comment]
I suspect that the number was added twice (once with spaces, once without) and that the system simply only shows you one of them. Are you using Google for your address book? Then you could take a look at https://contacts.google.com/, it might show you more details.

**Comment by ovalseven8** [comment]
No, I did not add the number twice.

**Comment by dbrgn** [comment]
Did you take a look at the Google contacts?

**Comment by ovalseven8** [comment]
My contact list is not synchronized with Google. So I can't find contacts at this website.

**Comment by dbrgn** [comment]
I could reproduce this with a contact. Will take a closer look.

**Comment by dbrgn** [comment]
This is a limitation of the Android address book. It combines multiple contacts. A contact is not necessarily only from your phone book, but also from other messaging apps.

In my case I have a contact where I only specified the number once (with spaces). But the Telegram app also provides a number for that contact, without spaces. And another messaging app (I don't know which one) provides the same number (with spaces) as the one I defined, but in the category "Other" than "Mobile".

Maybe we can deduplicate the numbers somehow. But probably not across categories.
